app/screens/EditLocationDetails.js

```diff
@@ -151,7 +151,6 @@ class EditLocationDetails extends Component {
                                         <Text style={s.title}>Phone</Text>
                                         <TextInput
                                             style={[{height: 40},s.textInput,s.radius10]}
-                                            keyboardType='numeric'
```

Comment:
This is a good fix. I think we should merge this for now, but it does open up the potential for a larger discussion about how to deal with phone numbers in general. Because the user now again _can_ add the dashes, they don't have to and if they don't, it looks weird on the location details screen. I don't know if we want to put any additional effort into this, especially since supporting international area codes adds complexity, but something we could continue to think on. We could consider some regex validating the phone number before we make the api request or possibly start storing just the numeric values in the db to allow us to format accordingly on the location details screen.

Reply:
Yeah… it’s really confusing once you add in international numbers. They’re all different! Definitely something to think/talk about in the future.

I recently created a one-time script for identifying the US numbers that have no spaces or dashes and inserting the dashes: https://github.com/scottwainstock/pbm/wiki/Clean-up-US-phone-numbers

Reply:
there's a thing from npm that would do a lot of this data normalization https://www.npmjs.com/package/phone

not a panacea but would work if international numbers are relevant... it would work regardless though.

Reply:
Thanks! We'll explore that. 

Just noting: on the website, we use the phonelib gem to validate phone numbers https://github.com/scottwainstock/pbm/search?q=phonelib
Though I don't think we're using this to validate location edits (either on the website or on the app/api) - if I'm remembering right, we're only using this to validate phone numbers in new location submissions (when we approve the submissions).

Reply:
ah yeah it would probably make sense to validate and normalize on the rails side of things.

Looking at the rails code it does look like the validations are happening every time the location is saved regardless of source.